Help/FAQ.rst

```diff
@@ -20,6 +20,23 @@ In short coala does two things:
   everything but the actual analyzer routine. Write just the parameters
   your custom analysis needs and the analysis part, we'll take care of the rest
 
+What is a bear anyway?
+----------------------
+
+A bear is meant to do some analysis on source code. It can check your code for
+potential problems, calculate metrics and even provide corrections for your
+code. The source code will be provided by coala so the bear doesn’t have to
```

Comment:
`The source code will be provided by coala...` This sentence isn't very clear. Can you maybe reword it?